### DISCUSSION TITLE
multicollinearity warning

### DISCUSSION THREAD

**Discussion by ashleyjthomas** [easystats/bayestestR#555] [Q&A]
I’m getting the following warning and I was wondering if this is something that might be ignored? 

Possible multicollinearity between b_Hierarchy_ZeroSum:other_MS_cooper and b_Intercept[2] (r = 0.86), b_Hierarchy_ZeroSum:other_MS_cooper and b_Intercept[3] (r = 0.87), b_Hierarchy_ZeroSum:other_MS_cooper and b_Intercept[4] (r = 0.87). This might lead to inappropriate results. See 'Details' in '?rope’.

This is the model that I am fitting (its likert scale data)

fit_MS_other <- brm( formula = plan_MS ~ Hierarchy_ZeroSum*other_MS_cooper+Identify+LogYears, 
data = TEST_data, 
family = cumulative("probit"),save_all_pars = TRUE)
![Screen Shot 2022-07-19 at 5 04 51 PM](https://user-images.githubusercontent.com/60658537/179871455-e732d31a-7dc9-4ca4-bf7e-8f80e8aa8cb3.png)

**Comment by bwiernik** [comment]
I wouldn't worry about this. It's just saying there is a strong separation between levels of your factor variables and each of the dependent variable scale points (which is why the intercept thresholds are correlated with the predictors)

@easystats/maintainers we should probably omit the intercept parameters from collinearity checks. Perhaps for ordinal models like this we should check the ordering of the intercepts

**Reply by DominiqueMakowski** [reply]
Yeah, also we should probably add some links in [ROPE's docs](https://easystats.github.io/bayestestR/reference/rope.html#multicollinearity-non-independent-covariates) to performance::check_model() and collinearity information
